Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,35 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
+        [TestMethod]
+        public void FileLogger_AppendsLogsToFile()
+        {
+            string message1 = "What is the average air speed velocity of an unladen swallow?";
+            string message2 = "African or European?";
+
+            if(File.Exists("fileLoggerTest.txt"))
```

Comment:
```suggestion
            string fileName = "fileLoggerTest.txt"
            if(File.Exists(fileName))
```
Since you are typing this multiple times, it may be better to store it in a variable you then reuse.